Replace http with https

puneet-pdx opened the pull request:
- update `http` to `https`
- update bintray references
- update `Release.md`

esreli commented on RELEASE.md:
```diff
@@ -1,3 +1,8 @@
+# Release 1.0.10
+
+- Updates doc to avoid all use of `http:`
```
This work makes changes to doc as well as source code. As such, I suggest we modify this line to read,

```suggestion
- Updates project to favor the use of secured `https` rather than `http`
```

puneet-pdx replied:
updated

esreli reviewed (changes requested):
The work captured in source and doc look good. Because this work will constitute a patch release, would you please update the app version to correlate with the stated release, `1.0.10`?

puneet-pdx commented:
> The work captured in source and doc look good. Because this work will constitute a patch release, would you please update the app version to correlate with the stated release, `1.0.10`?

@esreli I have addressed your comments.
